src/detectors/JavaScript/JavaScriptLanguageDetector.spec.ts

```diff
@@ -64,39 +47,27 @@ describe('JavaScriptLanguageDetector', () => {
 
   it('detects typescript correctly via ts file', async () => {
     const structure: VirtualDirectory = {
-      type: MetadataType.dir,
-      children: {
-        'index.ts': {
-          type: MetadataType.file,
-          data: '...',
-        },
-      },
+      '/index.ts': '...',
     };
 
     virtualFileSystemService.setFileSystem(structure);
 
     const langAtPath = await detector.detectLanguage();
     expect(langAtPath.length).toEqual(1);
     expect(langAtPath[0].language).toEqual(ProgrammingLanguage.TypeScript);
-    expect(langAtPath[0].path).toEqual('/');
+    expect(langAtPath[0].path).toEqual(nodePath.normalize('/'));
```

Comment:
I am also not sure what's the point of `normalize` here... Is it meant to replace `/` with `\` on Windows? I wonder whether it's a good idea because `normalize` does this just "by accident"... I would rather see `nodePath.sep`...

Reply:
I'll try on windows but I guess that Windows adds `C:\\` in normalize method and it is the reason why I didn't use just `nodePath.sep`.

Reply:
It works! Fixed